README.md

```diff
@@ -93,6 +93,23 @@ This gem also includes a dependency on `rubocop-rspec` that will be updated to
 the latest compatible version each time that the MAJOR.MINOR version of `rubocop` 
 is updated.
 
+## Change Process
+
+This configuration is meant to represent the general opinion of Salsify's Ruby community around best practices for
+writing readable Ruby code. As a result, changes to this configuration should go through a discussion phase in the
+#rubocop-changes Slack channel to ensure the broader Salsify Ruby community is on board with the change.
```

Comment:
Should we document what non-Salsify developers should do since this gem is open source (mainly so we can use it in our open source libraries)? I don't expect this to happen in practice but perhaps we should direct people to file a Github issue?